configs/components/windows_puppet.json

```diff
@@ -1,4 +1,4 @@
 {
   "url": "git://github.delivery.puppetlabs.net/puppetlabs-puppet_for_the_win.git",
-  "ref": "refs/tags/4.4.0.1"
+  "ref": "refs/tags/4.4.0"
```

Comment:
@Iristyle changes per your comment in https://tickets.puppetlabs.com/browse/PA-205